gp/node.py

```diff
@@ -22,14 +28,17 @@ class Node:
     """Base class for primitive functions used in Cartesian computational graphs.
     """
 
-    _arity = None
-    _active = False
-    _inputs = None
-    _output = None
-    _idx = None
-    _is_parameter = False
-
-    def __init__(self, idx, inputs):
+    __name__ = "Node"
+    _arity: int = 0
```

Comment:
again, this is suboptimal since the intention of setting this to `None` is to cause failures if it's not set. if it's set to `0` it could just generate nonsense results

Reply:
yes, fixed.